packages/web-sdk/src/transports/otlp/otlpPayloadLogger.ts

```diff
@@ -0,0 +1,3 @@
+import { createInternalLogger } from '@grafana/faro-core';
+
+export const internalLogger = createInternalLogger();
```

Comment:
Wrong usage of logger.

Internal logger is part of the `transport` class and provided via `BaseExtension`.

Will add this in follow and remove this logger.